src/test/java/org/foundation/coding/movie/NewReleaseMovieTest.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Refactor Code Tasks.
+ */
+package org.foundation.coding.movie;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * test class to new release movie.
+ */
+public class NewReleaseMovieTest {
+  public static final double RESULT = 30;
+  public static final int DAYS = 10;
+  public static final int DELTA = 0;
+
+  /**
+   * test method.
+   */
+  @Test
+  public void getFee() {
+    NewReleaseMovie newReleaseMovie = new NewReleaseMovie("Test", 1);
+    double resultExpected = RESULT;
+    double resultActual = newReleaseMovie.getFee(DAYS);
+    assertEquals(resultExpected, resultActual, DELTA);
+  }
+
+  /**
+   * test method.
+   */
+  @Test
+  public void getFrequentRenterPoints() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)